pkg/controller/networkpolicy/clustergroup_test.go

```diff
@@ -181,7 +211,11 @@ func TestAddClusterGroup(t *testing.T) {
 	selectorC := metav1.LabelSelector{MatchLabels: map[string]string{"foo3": "bar3"}}
 	selectorD := metav1.LabelSelector{MatchLabels: map[string]string{"foo4": "bar4"}}
 	cidr := "10.0.0.0/24"
+	exceptCIDR1, exceptCIDR2 := "10.0.0.0/25", "10.0.0.128/26"
 	controlplaneIPNet, _ := cidrStrToIPNet(cidr)
+	controlplaneIPNetExcept1, _ := cidrStrToIPNet(exceptCIDR1)
+	controlplaneIPNetExcept2, _ := cidrStrToIPNet(exceptCIDR2)
+	_, controlplaneIPNetDiff, _ := net.ParseCIDR("10.0.0.192/26")
```

Comment:
can we use a slightly more complicated case, where the computed `controlplaneIPNetDiff` is not a single cidr?
I notice that the except cidrs you have are either at the very beginning or at the very end of the IP block, which means we only ever have a single resulting cidr

Reply:
I've put a more extensive set of tests in `crd_utils_test.go`